documentation/ab-testing-with-google-optimize.md

```diff
@@ -0,0 +1,29 @@
+# A/B testing with Google Optimize
+
+## Setting up and a new redirect experiment
+- On the [Google Optimize console](https://optimize.google.com/) under the account,
+select a container for the environment you wish to run an experiment on
+
+- Create a new `Redirect Test` - this will allow to testing two different landing pages, with different URLs. 
+
+- Add the URL variants e.g `?variant=1` and `?variant=2`
+
+- Setup page targeting rules, the test will be applied on pages that match the target criteria. More details about targeting rules [here](https://support.google.com/optimize/answer/6283424?hl=en]).
+
+- Link the experiment to Google Analytics and setup Objectives. Custom objectives can track JS events. Create a new custom objective or use exiting events, e.g.:
+`vacancy_applied`
+
+```bash
+$(document).on('click', '.vacancy-apply-link', function() {
+    gtag('event', 'vacancy_applied');
+  }
+);
+```
+
+- Optimize should be installed for each existing containers, see [_gtag config file](https://github.com/DFE-Digital/teacher-vacancy-service/blob/develop/app/views/shared/_gtag.html.haml) If you wish to replace an existing container add the new container id to `GOOGLE_OPTIMIZE_ID` environment var in the corresponding environment. This will require Terraform deployment.
```

Comment:
This guidance seems separate to "Setting up and a new redirect experiment".  This might come under a "Creating a new container" guidance heading.  Not sure how useful that is thought 🤷‍♂ .

Reply:
I see what you mean, but this seems to be an important part of setting up a redirect test. Also, we can assume that if a new experiment has created in an existing container, it will work since this configuration has done. Made some changes for this be57ef3